utils/shared.js

```diff
@@ -784,7 +784,7 @@ const cleanSurveyData = (data) => {
     const admin = require('firebase-admin');
     
     Object.keys(data).forEach(key => {
-        if(data[key] === null) {
+        if(data[key] == null) {
```

Comment:
Should we use strick check here `data[key] === null || data[key] === undefined`?

Reply:
sounds good, updated